code/modules/power/singularity/collector.dm

```diff
@@ -47,6 +59,8 @@
 			var/gasdrained = min(powerproduction_drain*drainratio*delta_time,loaded_tank.air_contents.get_moles(GAS_PLASMA))
 			loaded_tank.air_contents.adjust_moles(GAS_PLASMA, -gasdrained)
 			loaded_tank.air_contents.adjust_moles(GAS_TRITIUM, gasdrained)
+			var/msg = "Plasma depleted, recommend replacing tank."
+			radio.talk_into(src, msg, radio_channel)
```

Comment:
This should be in the other part of the `if`/`else` check

Reply:
I'm pretty sure the other part is only for research mode.

Reply:
Done